frontend/src/App.vue

```diff
@@ -24,11 +24,10 @@ onBeforeMount(async () => {
 // Top level error handler
 onErrorCaptured((e: Error) => {
   const toast = useToast();
-  toast.add({
-    severity: 'error',
-    summary: 'Error',
-    detail: e.message, life: 3000
-  });
+  toast.error(
+    'Error',
+    e.message
+  );
```

Comment:
Style: Things like this can be collapsed into one line. We don't need to over-indent this, especially when the signature is asking for two arguments and not a destructured object anymore.